[jit] normalize `getMethod`

suo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #37474 [torchbind] Better error message when missing init.
* #37473 [jit] move torchbind tests to separate file
* **#37472 [jit] normalize `getMethod`**

Our convention is for `findX` to return an optional version and `getX`
to assert that the X is there. Fix up `getMethod` to be consistent with
this convention.

Differential Revision: [D21297543](https://our.internmc.facebook.com/intern/diff/D21297543)

suo committed:
[jit] normalize `getMethod`

Our convention is for `findX` to return an optional version and `getX`
to assert that the X is there. Fix up `getMethod` to be consistent with
this convention.

[ghstack-poisoned]

suo committed:
Update on "[jit] normalize `getMethod`"


Our convention is for `findX` to return an optional version and `getX`
to assert that the X is there. Fix up `getMethod` to be consistent with
this convention.

[ghstack-poisoned]

suo committed:
Update on "[jit] normalize `getMethod`"


Our convention is for `findX` to return an optional version and `getX`
to assert that the X is there. Fix up `getMethod` to be consistent with
this convention.

Differential Revision: [D21297543](https://our.internmc.facebook.com/intern/diff/D21297543)

[ghstack-poisoned]

eellison commented:
I think you're going to have to rebase this past https://github.com/pytorch/pytorch/pull/37424, which is landing now.

suo committed:
Update on "[jit] normalize `getMethod`"


Our convention is for `findX` to return an optional version and `getX`
to assert that the X is there. Fix up `getMethod` to be consistent with
this convention.

Differential Revision: [D21297543](https://our.internmc.facebook.com/intern/diff/D21297543)

[ghstack-poisoned]

suo committed:
Update on "[jit] normalize `getMethod`"


Our convention is for `findX` to return an optional version and `getX`
to assert that the X is there. Fix up `getMethod` to be consistent with
this convention.

Differential Revision: [D21297543](https://our.internmc.facebook.com/intern/diff/D21297543)

[ghstack-poisoned]

suo committed:
Update on "[jit] normalize `getMethod`"


Our convention is for `findX` to return an optional version and `getX`
to assert that the X is there. Fix up `getMethod` to be consistent with
this convention.

Differential Revision: [D21297543](https://our.internmc.facebook.com/intern/diff/D21297543)

[ghstack-poisoned]
